css/index.css

```
@@ -1,102 +1,270 @@
 }
 
-.carousel-food-spin__dish:nth-child(8) {
-  top: 250px;
-  left: 50px;
+.food-spin__order-button:hover {
+  background-color: #ff6600;
 }
 
-.carousel-food-spin__dish.active ~ .carousel-food-spin__dish {
-  opacity: 0.5;
+.food-spin__video{
+  font-size: var(--font-size-smalln);
+  font-weight: var(--font-weight);
+  line-height: var(--twenty-pixel);
+  text-align: left;
+  margin-top: 142px;
 }
 
-.carousel-food-spin__controls {
+.food-spin__rigth {
+  min-height: 90vh;
+  margin-top: 56px;
+  flex-basis: 950px;
+}
+
+.slide-food-spin {
+  position: relative;
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+.carousel-food-spin {
+  position: relative;
+  padding-top: 100%;
+}
+
+.carousel-food-spin__dishs {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  padding-left: 0px;
+  margin: 0px;
+  list-style-type: none;
+}
+
+.carousel-food-spin__dish {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  opacity: 0;
+}
+
+
+.carousel-food-spin__dish.active {
+  z-index: 1;
+  opacity: 1;
+}
+
+.carousel-food-spin__snip{
+  clip-path: circle(149% at 0% -148%);
+  z-index: 3;
   position: absolute;
   top: 50%;
   left: 50%;
-  transform: translate(-50%, -50%);
-  display: flex;
+  width: 100%;
+  height: 100%;
 }
 
-.carousel-food-spin__control {
-  background-color: #ff9800;
-  color: #fff;
-  border: none;
-  padding: 10px 18px;
+.carousel-food-spin__pagination{
+  width: 90%;
+  height: 90%;
+  border-radius: 50%;
+  border: 2px dashed #E87F1E;
+  transition-property: transform;
+  transition-timing-function: ease-out;
+  pointer-events: none;
+  user-select: none;
+  list-style-type: none;
+  padding-left: 0px;
+  margin: 0px;
+}
+
+.carousel-food-spin__item{
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+}
+
+.carousel-food-spin__dot {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translate(-50%, -50%);
   border-radius: 50%;
-  margin: 0 10px;
+  pointer-events: auto;
+  transition: 0.2s;
+}
+
+.carousel-food-spin__next,
+.carousel-food-spin__prev{
+  display: flex;
+  align-items: center;
+  position: absolute;
+  bottom: 13%;
+  z-index: 1;
+  letter-spacing: 0.3em;
+  transition: 0.3s;
   cursor: pointer;
-  font-size: 20px;
-}
\ No newline at end of file
+  padding: 14px;
+  border-radius: 50%;
+  background-color: #FF922C;
+  box-shadow: 0px 10px 20px 0px #F5ECE3;
+  transition: background-color 0.3s ease;
+  border: none;
+}
+
+.carousel-food-spin__next {
+  right: -10%;
+}
+.carousel-food-spin__prev {
+  left: -10%;
+}
+
+.carousel-food-spin__main {
+  position: absolute;
+  top: 57%;
+  margin-left: 250px;
+  transform: translate(-50%, -50%);
+  display: flex;
+  z-index: 10;
+}
+
+.carousel-food-spin__img {
+  width: 290px;
+  margin: 0px;
+}
+
+.carousel-food-spin__img1{
```

Creo que es mas elegante ponerle el nombre del padre que un simple `1`
```suggestion
.carousel-food-spin__item-img {
```